widgetastic_manageiq/__init__.py

```diff
@@ -4437,3 +4437,150 @@ def fill(self, value):
             raise TypeError("Fill can only take one value from list to click on.")
         self.select_plan(value)
         return True
+
+
+class InfraMappingList(Widget):
+    """Represents the list of Infrastructure Mappings."""
+
+    ROOT = ParametrizedLocator('.//div[contains(@class,{@list_class|quote})]')
+    ITEM_LOCATOR = ParametrizedLocator('./div[contains(@class,"list-group-item")]')
```

Comment:
Why is this `ParametrizedLocator`? Doesn't look like it needs to be.

Reply:
good catch. ;)